docs/_data/sidebar.yml

```diff
@@ -27,10 +19,6 @@
         en: Tips & Tricks
         ja: ヒントとコツ
       url: /getting-started/tips-and-tricks
-    - title:
```

Comment:
Does this mean we should delete the `java-setup.md` files or how are we linking to them if it's not through the sidebar ?

Reply:
Nevermind about this, I see it got added at the end of install.md